### DISCUSSION TITLE
Adding Bot to Server

### DISCUSSION THREAD

**Discussion by stear541** [DMcP89/harambot#53] [Q&A] [answered]
Hi there, I have the bot running locally and see the following message in the WSL terminal:
[2022-09-13 12:46:46,357 INFO] [harambot.py.harambot.on_ready] Everything's all ready to go~
[2022-09-13 13:16:58,971 INFO] [harambot.py.harambot.on_guild_join] Joined The Homies Fantasy Football

So the bot is running and has been added to the server successfully.

However commands within Discord are not quite working.  Typing `$standings` for example does not return any message from the bot.  What permissions are needed on the bot's OAuth2 URL generator?  I have bot selected followed by Send Messages and Embed Links.  Anything else needed?

Any other tips?

Thanks!

**Comment by DMcP89** [comment]
@stear541 can you run the $ping command and tell me if that returns anything? Also if you can provide the logs from your bot.

**Comment by DMcP89** [comment]
@stear541  I have updated the readme file to include what permissions to give the bot. Please let me know if this helps!

**Comment by DMcP89** [comment, marked as answer]
Please see #61  for resolution
